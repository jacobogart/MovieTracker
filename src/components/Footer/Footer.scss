.Footer {
  display: flex;

  .footer-wrapper {
    padding: 30px 150px;

    .footer-credit {
      height: 60px;
    }
  }
}