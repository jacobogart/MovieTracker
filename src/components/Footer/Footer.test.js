import Footer from "./Footer";
import { shallow } from 'enzyme';
import React from 'react';

describe('Footer', () => {
  it('should match the snapshot', () => {
    let wrapper = shallow(<Footer />);
    expect(wrapper).toMatchSnapshot();
  });
});