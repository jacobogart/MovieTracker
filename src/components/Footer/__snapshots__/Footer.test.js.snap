// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should match the snapshot 1`] = `
<section
  className="Footer"
>
  <div
    className="footer-wrapper"
  >
    <a
      href="https://www.themoviedb.org/"
    >
      <img
        alt="Logo"
        className="footer-credit"
        src="https://www.themoviedb.org/assets/2/v4/logos/408x161-powered-by-rectangle-green-bb4301c10ddc749b4e79463811a68afebeae66ef43d17bcfd8ff0e60ded7ce99.png"
      />
    </a>
  </div>
</section>
`;
